admin_guide/managing_pods.adoc

```diff
@@ -146,16 +146,126 @@ endif::openshift-enterprise,openshift-origin[]
-using either a xref:admin-guide-deploying-an-egress-router-service[router] or
-xref:admin-guide-limit-pod-access-egress[firewall] methods.
+As an {product-title} cluster administrator, you can control egress traffic in two ways:
+
+xref:admin-guide-limit-pod-access-egress[Firewall]::
+Using an egress firewall allows you to enforce the acceptable outbound traffic
+policies, so that specific endpoints or IP ranges (subnets) are the only
+acceptable targets for the dynamic endpoints (pods within {product-title}) to
+talk to.
+
+xref:admin-guide-limit-pod-access-egress-router[Router]::
+Using an egress router allows you to create identifiable services to send
+traffic to a specific source, ensuring an external source treats traffic as
+though it were coming from a known source. This helps with security, because it
+allows you to secure an external database so that only specific pods in a
+namespace can talk to a service (the egress router), which proxies the traffic
+to your database.
+
+
+[[admin-guide-limit-pod-access-egress]]
+=== Limiting Pod Access with Egress Firewall
+
+As an {product-title} cluster administrator, you can use egress firewall policy
+to limit the external addresses that some or all pods can access from within the
+cluster, so that:
+
+- A pod can only talk to internal hosts, and cannot initiate connections to the
+public Internet.
++
+Or,
+- A pod can only talk to the public Internet, and cannot initiate connections to
+internal hosts (outside the cluster).
++
+Or,
+- A pod cannot reach specified internal subnets/hosts that it should have no
+reason to contact.
+
+You can configure projects to have different egress policies. For example,
+allowing `<project A>` access to a specified IP range, but denying the same
+access to `<project B>`. Or restrict application developers from updating from
+(python) pip mirrors, and forcing updates to only come from desired sources.
+
+[CAUTION]
+====
+You must have the
+xref:../install_config/configuring_sdn.adoc#install-config-configuring-sdn[*ovs-multitenant* plug-in] enabled in order to limit pod access via egress policy.
+====
+
+Project administrators can neither create `EgressNetworkPolicy` objects, nor
+edit the ones you create in their project. There are also several other
+restrictions on where `EgressNetworkPolicy` can be created:
+
+* The `default` project (and any other project that has been made global via
+`oadm pod-network make-projects-global`) cannot have egress policy.
+
+* If you merge two projects together (via `oadm pod-network join-projects`),
+then you cannot use egress policy in _any_ of the joined projects.
+
+* No project may have more than one egress policy object.
+
+Violating any of these restrictions results in broken egress policy for the
+project, and may cause all external network traffic to be dropped.
+
+Use the `oc` command or the REST API to configure pod access limits. You can use
```

Comment:
Probably it's best to get rid of the "pod access limits" language altogether? "to configure egress policy" (and then again below)